spec/index.bs

```diff
@@ -2502,6 +2534,8 @@ GPUQueue includes GPUObjectBase;
 {{GPUQueue/submit(commandBuffers)}} does nothing and produces an error if any of the following is true:
 
  - Any {{GPUBuffer}} referenced in any element of `commandBuffers` isn't in the `"unmapped"` [=buffer state=].
+ - Any of the [=usage scopes=] contained in the command buffers fail the [=usage scope validation=].
```

Comment:
```suggestion
 - Any of the [=usage scopes=] contained in the command buffers fail [=usage scope validation=].
```

Reply:
I wonder why `the` shouldn't be here?

Reply:
I agree it shouldn't be there, but kind of okay either way: "if any X fail validation." vs "if any X fail the validation."